### PR TITLE
feat(free-courses-es.md): new free courses in Spanish have been added

### PR DESCRIPTION
**Pull request by Jeremy-Aza**
## What does this PR do?
Add a free course on youtube and web platforms for git & github, javascript, react native and design patterns in spanish.

### Description
These are two long youtube videos and two platforms to learn design patterns, javascript, git & github and react native in spanish.
The two videos do not constitute a playlist. They are individual videos that are structured as an episodic course and have a duration of 4 to 5 hours.
### Why is this valuable (or not)?
Because they are interactive, with complementary examples that help to improve understanding.
### How do we know it's really free?
They are on youtube and free access platforms. 
### For book lists, is it a book? For course lists, is it a course? etc.
These are courses
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-es.md [hunk 1]
```diff
@@ -84,6 +85,7 @@
 
 ### Control de Versiones
 
+* [Curso de GIT y GITHUB desde Cero para principiantes](https://www.youtube.com/watch?v=3GymExBkKjE&ab_channel=MoureDevbyBraisMoure) - Brias Moure (MoureDev)
```
Comment (eshellman) [inline courses/free-courses-es.md:88]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-es.md [hunk 2]
```diff
@@ -146,6 +148,11 @@
 * [Word Avanzado (2010)](https://www.pildorasinformaticas.es/course/word-avanzado-2010) - Juan Díaz (Píldoras Informáticas)
 
 
+### Patrones de diseño
+
+* [Patrones de diseño](https://refactoring.guru/es/design-patterns) - Alexander Shvets (Refactoring.Guru)
```
Comment (eshellman) [inline courses/free-courses-es.md:153]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.
belongs in Programación I think

File: courses/free-courses-es.md [hunk 3]
```diff
@@ -158,6 +165,7 @@
 ### Programación
 
 * [Aprende JavaScript](https://aprendejavascript.org) - Jonathan MirCha
+* [Aprende javascript](https://www.aprendejavascript.dev) - Miguel Ángel Durán (midudev)
```
Comment (eshellman) [inline courses/free-courses-es.md:168]:
```suggestion
* [Aprende javascript](https://www.aprendejavascript.dev) - Miguel Ángel Durán
```

File: courses/free-courses-es.md [hunk 4]
```diff
@@ -206,6 +214,7 @@
 * [Curso gratuito de JavaScript](https://argentinaprograma.com) - Fabricio Sodano (Argentina Programa)
 * [Curso gratuito de Next.js y Firebase](https://www.youtube.com/playlist?list=PLV8x_i1fqBw1VR86y4C72xMGJ8ifjBwJ6) - Miguel Ángel Durán «midudev»
 * [Curso JSON. De Novato a Experto](https://www.youtube.com/playlist?list=PLrDTf5qnZdEAiHO19QB9hq5QXAef1h8oY) - Camilo Martínez "Equimancho"
+* [CURSO REACT NATIVE desde CERO](https://www.youtube.com/watch?v=qi87b6VcIHY&t=1004s&ab_channel=midudev) - Miguel Ángel Durán (midudev)
```
Comment (eshellman) [inline courses/free-courses-es.md:217]:
```suggestion
* [Curso React Native desde CERO](https://www.youtube.com/watch?v=qi87b6VcIHY) - Miguel Ángel Durán (midudev)
```
Is CERO an abbreviation? We don't use ALL CAPS title except for abbreviations.

**Review by eshellman — CHANGES_REQUESTED** [review]
as you can see by other listings here, there's a split on how to attribute midudev. For youtube, we generally use the channel name to make it easier to spot duplicates. we dislike using both channel name and instructor name. for non-youtube resources we don't use a channel name.
For midudev, if it's a nickname `Miguel Ángel Durán "midudev"` would be best. Or just `midudev`

File: courses/free-courses-es.md (hunk 3 above)
Comment (Jeremy-Aza) [reply courses/free-courses-es.md]: Thanks!!

File: courses/free-courses-es.md (hunk 1 above)
Comment (Jeremy-Aza) [reply courses/free-courses-es.md:88]: Yes, but this video is over an hour long and is structured as an episodic course

File: courses/free-courses-es.md (hunk 4 above)
Comment (Jeremy-Aza) [reply courses/free-courses-es.md]: I'm Sorry. The word CERO is not an abbreviation and a correction to the title has already been uploaded

File: courses/free-courses-es.md (hunk 2 above)
Comment (Jeremy-Aza) [reply courses/free-courses-es.md]: You are right. I have already placed this resource in the programming category

File: courses/free-courses-es.md (hunk 1 above)
Comment (eshellman) [reply courses/free-courses-es.md]: OK. When this is the case, use the PR description to let us know.

File: courses/free-courses-es.md [hunk 5]
```diff
@@ -206,6 +209,7 @@
 * [Curso gratuito de JavaScript](https://argentinaprograma.com) - Fabricio Sodano (Argentina Programa)
 * [Curso gratuito de Next.js y Firebase](https://www.youtube.com/playlist?list=PLV8x_i1fqBw1VR86y4C72xMGJ8ifjBwJ6) - Miguel Ángel Durán «midudev»
 * [Curso JSON. De Novato a Experto](https://www.youtube.com/playlist?list=PLrDTf5qnZdEAiHO19QB9hq5QXAef1h8oY) - Camilo Martínez "Equimancho"
+* [Curso React Native desde cero](https://www.youtube.com/watch?v=qi87b6VcIHY&t=1004s&ab_channel=midudev) - Miguel Ángel Durán
```
Comment (eshellman) [inline courses/free-courses-es.md:212]:
```suggestion
* [Curso React Native desde cero](https://www.youtube.com/watch?v=qi87b6VcIHY&t=1004s) - Miguel Ángel Durán
```

**Review by eshellman — CHANGES_REQUESTED** [review]
Are the other midudev resources labeled the same way? (can deal with that in another PR)

File: courses/free-courses-es.md (hunk 1 above)
Comment (Jeremy-Aza) [reply courses/free-courses-es.md]: Thank you for your comments. I have already added this information from the description

**Comment by Jeremy-Aza** [comment]
> Are the other midudev resources labeled the same way? (can deal with that in another PR)

No, the other midudev resources are not labeled that way

**Comment by eshellman** [comment]
OK, I still like `Miguel Ángel Durán "midudev"` - that can be another PR
Thanks, and welcome to free-programming-books!

**Comment by Jeremy-Aza** [comment]
Thanks for all. I created a new PR to add nickname "midudev"

**Pull request merged**